Nos/Views/New Note/NewNoteView.swift

```diff
@@ -175,6 +183,22 @@ struct NewNoteView: View {
             if let expirationTime {
                 tags.append(["expiration", String(Date.now.timeIntervalSince1970 + expirationTime)])
             }
+            
+            // Attach the new note to the one it is replying to, if any.
+            if let replyToNote = replyTo, let replyToNoteID = replyToNote.identifier {
```

Comment:
Yeah, I liked adding the further context that `replyTo` is a `Note`, as it makes its usage below more clear to me.

Reply:
Yeah, that makes sense. I'd probably prefer the property to also be called `replyToNote` but it's not a dealbreaker for me.

Reply:
Although when I reread the comment on it, I'm leaning heavily that direction:

```The note that the user is replying to, if any.```

To me it's much more clear if it's called `replyToNote` since its type is actually `Event?` and not all Events are notes.